x-pack/plugins/security_solution/cypress/integration/value_lists.spec.ts

```diff
@@ -39,6 +46,7 @@ describe('value lists', () => {
     });
 
     afterEach(() => {
+      removeSignalsIndex();
```

Comment:
If all our tests use `cleanKibana` in a `beforeEach`, I don't think we need the same in `afterEach`.

However, that does bring up a general question about teardown: what is the cypress suite's teardown responsibility as a whole? Is cypress relegated to its own CI group and so we don't have to worry about cleaning up for the "next" suite? Is there a possibility that tests will be added to the cypress group and we _do_ need to start caring about this?

I'm thinking about the situations that have bit us in the past, namely incomplete teardown and shared state, which are strongly related. We should do whatever we can to minimize those and isolate tests.